app/controllers/scores_controller.rb

```diff
@@ -12,8 +12,9 @@ def create
   end
 
   def update
-    flash.now[:notice] = 'Your changes were saved!'
-    redirect_to users_dashboard_index_path
+    respond_to do |format|
+      format.js { flash.now[:notice] = 'Your changes were submitted!'}
```

Comment:
This is the format that rails doesn't like, except that it works in local host. I looked into passing a format along. The syntax allows me to pass the message along

Reply:
another annoying thing was that what the test liked, didn't work in local host so go figure there